docs/api/registration-base-url-resource.md

```diff
@@ -225,7 +225,7 @@ Name             | Type             | Required | Notes
 ---------------- | ---------------- | -------- | -----
 reasons          | array of strings | yes      | The reasons why the package was deprecated
 message          | string           | no       | The additional details about this deprecation
-alternatePackage | object           | no       | The package dependency that should be used instead
+alternatePackage | object           | no       | The [package dependency](#package-dependency) that should be used instead
```

Comment:
An alternate package can have a `*` range. A package dependency cannot.

Reply:
😓

Reply:
I called this out for both package dependency and alternate package objects. Personally, I feel iffy about calling out that this is supported for `alternatePackage`. Should I remove it?

Reply:
I think the docs should at the minimum mention something about alternate package + `*` because it's there today. Prescriptions or semantics can be any level of vague IMO.